### PR TITLE
Bump version to 0.16.0 #WIP

### PR DESCRIPTION
**Pull request by yescallop**
From https://github.com/pmmp/PocketMine-MP/tree/0.16
Known problems:
 - OutOfMemoryError

WIP - This implementation is not functional yet, the game is not joinable at present. Just wait in peace, if you need it now, you can contribute to our work -- Snake1999

### CODE REVIEW

**Comment by File14** [comment]
Have you tested it? Is it joinable?

**Comment by Pub4Game** [comment]
@File14 no

File: src/main/java/cn/nukkit/Player.java [hunk 1]
```diff
@@ -1370,7 +1377,10 @@ public boolean setMotion(Vector3 motion) {
             if (this.chunk != null) {
                 this.level.addEntityMotion(this.chunk.getX(), this.chunk.getZ(), this.getId(), this.motionX, this.motionY, this.motionZ);
```
Comment (NycuRO) [inline src/main/java/cn/nukkit/Player.java]: Error here @yescallop

**Comment by Snake1999** [comment]
Moved to #1193

**Pull request closed without merging**